### PR TITLE
Update messages.json (ru)

### PR DESCRIPTION
**Pull request by ViktorOn**

### CODE REVIEW

File: src/_locales/ru/messages.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		"message": "Фоновый скрипт не отвечает"
 	},
 	"message_translateOfText": {
-		"message": "Перевод"
+		"message": "Переведено"
```
Comment (vitonsky) [inline src/_locales/ru/messages.json:30]: This key in not used. TODO: remove it

File: src/_locales/ru/messages.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
 		"message": "Сбросить настройки?"
 	},
 	"message_cacheFlushed": {
-		"message": "Кэш очищен"
+		"message": "Кеш очищен"
```
Comment (vitonsky) [inline src/_locales/ru/messages.json:45]: Remove this change https://ru.wikipedia.org/wiki/%D0%9A%D1%8D%D1%88

File: src/_locales/ru/messages.json [hunk 3]
```diff
@@ -96,13 +96,13 @@
 		"message": "Словарь пуст. Начните сохранять тексты при переводе"
 	},
 	"dictionary_notFoundEntries": {
-		"message": "Не найдены переводы удовлетворяющие фильтрам"
+		"message": "Не найдены записи, соответствующие фильтрам"
```
Comment (vitonsky) [inline src/_locales/ru/messages.json:99]: Loosed context. Remove it

File: src/_locales/ru/messages.json [hunk 4]
```diff
@@ -114,7 +114,7 @@
 		"message": "Очистить"
 	},
 	"dictionary_filter_from": {
-		"message": "Оригинальный язык"
+		"message": "Исходный язык"
```
Comment (vitonsky) [inline src/_locales/ru/messages.json:117]: I don't agree with this word, let's remove it

**Review by vitonsky — CHANGES_REQUESTED** [review]
Thanks for your contribution! Let's remove few changes

File: src/_locales/ru/messages.json (hunk 4 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:117]: I'm sorry, i've missed context. I thought it's for translate page, but for dictionary it's exact word!

File: src/_locales/ru/messages.json (hunk 3 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:99]: We may replace it to "Не найдены переводы, соответствующие фильтрам". I agree that "match" meaning is better than "satisfy", but we have to keep object of description - "translations", but not "entries"

File: src/_locales/ru/messages.json (hunk 1 above)
Comment (ViktorOn) [reply src/_locales/ru/messages.json:30]: ["message": "Translated"](https://github.com/translate-tools/linguist/blob/dd62177b28214151a39caee3d64edfbe5b0be84d/src/_locales/en/messages.json#L30)

File: src/_locales/ru/messages.json (hunk 2 above)
Comment (ViktorOn) [reply src/_locales/ru/messages.json:45]: image

File: src/_locales/ru/messages.json (hunk 3 above)
Comment (ViktorOn) [reply src/_locales/ru/messages.json:99]: > Loosed context. Remove it

I always refer to the [English original document](https://github.com/translate-tools/linguist/blob/dd62177b28214151a39caee3d64edfbe5b0be84d/src/_locales/en/messages.json#L99)

File: src/_locales/ru/messages.json (hunk 1 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:30]: It's not for you, it's note for me

File: src/_locales/ru/messages.json (hunk 2 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:45]: The word is already used. It's about preferences, but not about correct meaning. Let's avoid grammar nazi changes

Comment (ViktorOn) [reply src/_locales/ru/messages.json:45]: But what about the next translation? It uses "[кеш](https://github.com/translate-tools/linguist/blob/dd62177b28214151a39caee3d64edfbe5b0be84d/src/_locales/ru/messages.json#L225)"

File: src/_locales/ru/messages.json (hunk 3 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:99]: Thanks, we have to fix it there too. You may do it in your PR if you want

**Comment by ViktorOn** [comment]
Basically I don't understand what the problem is...

File: src/_locales/ru/messages.json (hunk 2 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:45]: Agreed, your fix improve text style cosistency

File: src/_locales/ru/messages.json (hunk 3 above)
Comment (vitonsky) [reply src/_locales/ru/messages.json:99]: Let's fix in document mentioned you above to replace "Entries" to "Translations" and will merge

**Comment by vitonsky** [comment]
Thanks for contribution

**Pull request merged**